vmm/src/lib.rs

```diff
@@ -998,7 +1041,40 @@ impl Vmm {
             }
 
             // Send config
-            let config_data = serde_json::to_vec(&vm.get_config()).unwrap();
+            let vm_config = vm.get_config();
+            #[cfg(all(feature = "kvm", target_arch = "x86_64"))]
+            let common_cpuid = {
+                #[cfg(feature = "tdx")]
+                let tdx_enabled = vm_config.lock().unwrap().tdx.is_some();
+                let phys_bits = vm::physical_bits(
+                    vm_config.lock().unwrap().cpus.max_phys_bits,
+                    #[cfg(feature = "tdx")]
+                    tdx_enabled,
+                );
+                arch::generate_common_cpuid(
```

Comment:
Don't we have access to this instead of regenerating it from scratch?

Reply:
Ditto.